tests/sqlparser_snowflake.rs

```diff
@@ -9,9 +9,26 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+#[macro_use]
+#[path = "utils/mod.rs"]
+mod utils;
+
 use sqlparser::ast::*;
 use sqlparser::dialect::{GenericDialect, SnowflakeDialect};
+use sqlparser::parser::ParserError;
 use sqlparser::test_utils::*;
+use utils::*;
+
+fn table_alias(alias: &str) -> TableAlias {
```

Comment:
Maybe keep `table(name)` as is to avoid changes to existing tests, and add a `table_with_alias(table, alias)` instead?

Reply:
Done